updated_main.py

```diff
@@ -3,57 +3,86 @@
 import bs4
 import time
 import json
+import os
+from email.message import EmailMessage
+import ssl
+import smtplib
 
 # Running a for loop to get the latest price of the GBP
 for i in range(100):
 
     # Now extracting the value of British Pound in Indian Rupees
-    currency = "convert 1 pound to rs"
+    def Getting_current_price():
+        currency = "convert 1 pound to rs"
+        url = "https://google.com/search?q=" + currency
+        request_result = requests.get(url)
 
-    url = "https://google.com/search?q=" + currency
+        soup = bs4.BeautifulSoup(request_result.text, "html.parser")
+        temp = soup.find("div", class_="BNeawe iBp4i AP7Wnd").text
+        temps = temp.split(" ")
+        current_price = float(temps[0])
+    Getting_current_price()
```

Comment:
Function name should always start from lowercase. All class names starts with uppercase.